samri/fetch/templates.py

```diff
@@ -1,14 +1,17 @@
 from sklearn.datasets.base import Bunch
 from nilearn.datasets.utils import _fetch_files
 
+import subprocess
 from os import path
 
-def fetch_rat_waxholm(data_dir="~/.samri_files/templates/rat/", verbose=1):
+def fetch_rat_waxholm(template_dir="~/.samri_files/templates/rat/waxholm/", verbose=1):
 	"""Download and load waxholm atlas for Sprague Dawley rat
 
 	Returns:
-	data:	sklearn.datasets.base.Bunch
-		Dictionary-like object, interest attributes are:
+        data:   sklearn.datasets.base.Bunch Dictionary-like object
+		template
```

Comment:
This description should be more informative.

Reply:
elaborated